docs/source/contribute.rst

```diff
@@ -75,9 +69,9 @@ Unless you are a U.S. Government Employee (see below), you will need to `agree t
 our Individual Contributor Agreement <http://www.projectcalico.org/community/individual-contributor-agreement/>`__
 before we can accept your contribution.
 
-In addition, if you are contributing on behalf of a company or 
+In addition, if you are contributing on behalf of a company or
 organization someone at your organization needs to sign the `Corporate Contributor Agreement <http://www.projectcalico.org/community/corporate-contributor-agreement/>`__.
 
-US Government employees should not sign the Individual Contributor 
-greement.  Instead, someone with authority to sign on behalf of your 
+US Government employees should not sign the Individual Contributor
+greement.  Instead, someone with authority to sign on behalf of your
```

Comment:
Clearly my fault from the original commit, but this seems to have lost the "A" on agreement.